account_fiscal_position_vies_warning/tests/common.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2025 Moduon Team S.L. <info@moduon.team>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests import TransactionCase
+
+
+class TestViesWarningCommon(TransactionCase):
```

Comment:
Use BaseCommon as base test class to reduce overhead from tracking and boost the test suite.

```suggestion
class TestViesWarningCommon(BaseCommon):
```